src/renderer/widgets/Transfer/default/model/form-model.ts

```diff
@@ -304,9 +298,79 @@ const { $errors, $valid } = createTxValidationStore({
     transaction: $calculationTx,
     xcmFee: xcmTransferModel.$xcmFee,
     deliveryFee: xcmTransferModel.$deliveryFee,
+    includeExistentialDeposit: $isExistentialDepositEnabled,
   },
 });
 
+const $totalFee = combine(
+  {
+    validationDone: $validationDone,
+    validationResults: $balanceValidationResults,
+    asset: $asset,
+  },
+  ({ validationResults, asset, validationDone }) => {
+    if (!validationDone || !asset) {
+      return null;
+    }
+
+    return combineTotalRequiredFee({ validationResults, assetId: asset.assetId, excludeActions: ['sending amount'] });
+  },
+);
+
+const $initiatorAccountBalance = combine(
+  {
+    initiator: form.fields.initiator.$value,
+    balances: balanceModel.$balanceMap,
+    chain: $chain,
+    asset: $asset,
+  },
+  ({ initiator, asset, chain, balances }) => {
+    if (nullable(initiator) || nullable(chain) || nullable(asset)) {
+      return null;
+    }
+
+    return balanceUtils.getBalance(balances, initiator.accountId, chain.chainId, asset.assetId);
+  },
+);
+
+const $available = combine(
+  {
+    balance: $initiatorAccountBalance,
+    chain: $chain,
+    asset: $asset,
+    totalFee: $totalFee,
+    isExistentialDepositEnabled: $isExistentialDepositEnabled,
+  },
+  ({ asset, chain, balance, totalFee, isExistentialDepositEnabled }) => {
+    if (nullable(balance) || nullable(chain) || nullable(asset) || nullable(totalFee)) {
+      return null;
+    }
+
+    return getAvailableAmount({ balance, totalFee, includeED: isExistentialDepositEnabled });
+  },
+);
+
+const $showAccountDeathAlert = combine(
+  {
+    isMaxMode: $isMaxModeEnabled,
+    balance: $initiatorAccountBalance,
+    totalFee: $totalFee,
+    amount: form.fields.amount.$value,
+    asset: $asset,
+  },
+  ({ isMaxMode, balance, totalFee, amount, asset }) => {
+    if (!isMaxMode || !totalFee || !asset) {
```

Comment:
we added burned amount in balance validation results especially for this case. Just pick balanceValudationResults and sum all burned tokens, if it's > 0 - show modal